internal/common/literals.go

```diff
@@ -24,10 +24,19 @@ func (lit *BasicLit) Value(vars map[string]interface{}) interface{} {
 	switch lit.Type {
 	case scanner.Int:
 		value, err := strconv.ParseInt(lit.Text, 10, 32)
-		if err != nil {
-			panic(err)
+		if err == nil {
+			return int32(value)
+		}
+		if numError, ok := err.(*strconv.NumError); ok {
+			if numError.Err == strconv.ErrRange {
+				val, e := strconv.ParseFloat(lit.Text, 64)
+				if e != nil {
+					panic(e)
+				}
+				return val
+			}
 		}
-		return int32(value)
+		panic(err)
```

Comment:
Is it possible to use `fallthrough` instead? Also, I'd prefer the happy paths to be last, for example:
```go
if err != nil {
	if nerr, ok := err.(*strconv.NumError); ok && nerr.Err == strconv.ErrRange {
		fallthrough // go to the parse float case
	}
	panic(err)
}
return int32(value)
```